android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/email/Email.kt

```diff
@@ -261,7 +264,11 @@ class Email {
     fun flush(context: Context, onComplete: () -> Unit, onError: (msg: String) -> Unit) {
         this.client(context, onError) { client ->
             client.logout().addOnSuccessListener {
-                onComplete()
+                client.clearLastCheckedTime().addOnSuccessListener {
```

Comment:
remove the callbacks. Call onComplete after it is all done. onError should not be triggered.